src/Parser/RetributionPaladin/Modules/Items/ChainOfThrayn.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import { formatNumber } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const CHAIN_OF_THRAYN_INCREASE = 0.1;
+
+class ChainOfThrayn extends Module {
+	static dependencies = {
+		combatants: Combatants,
+	};
+
+	damageDone = 0;
+
+	on_initialized() {
+		this.active = this.combatants.selected.hasWaist(ITEMS.CHAIN_OF_THRAYN.id);
+	}
+
+	on_byPlayer_damage(event) {
+		if(this.combatants.selected.hasBuff(SPELLS.CRUSADE_TALENT.id) || this.combatants.selected.hasBuff(SPELLS.CRUSADE_TALENT.id)){
+			this.damageDone += ((event.amount || 0) + (event.absorbed || 0)) * CHAIN_OF_THRAYN_INCREASE / (1 + CHAIN_OF_THRAYN_INCREASE);
```

Comment:
You are checking for the `CRUSADE_TALENT.id` twice here, did you mean to select a different spell?

Also, isn't the 10% bonus from Thrayn already in the damage event? If so the formula would be `bonusDamage = raw - (raw / (1 + Increase))` I believe.